cmd/immuadmin/command/backup.go

```diff
@@ -189,6 +189,23 @@ func offlineBackup(src string, uncompressed bool, manualStopStart bool) (string,
 		return "", fmt.Errorf("%s is not a directory", src)
 	}
 
+	currDir, err := os.Getwd()
+	if err != nil {
+		return "", err
+	}
+	srcAbs, err := filepath.Abs(src)
+	if err != nil {
+		return "", err
+	}
+	currDirAbs, err := filepath.Abs(currDir)
+	if err != nil {
+		return "", err
+	}
+	if srcAbs == currDirAbs {
```

Comment:
Perhaps the check can be done before the user is asked to authenticate.

Reply:
Changed in the latest commit. 👍